cmd/fs-v1.go

```diff
@@ -103,6 +103,11 @@ func newFSObjectLayer(fsPath string) (ObjectLayer, error) {
 		}
 	}
 
+	// Check if disk has minimum required total space.
+	if err = checkDiskTotalMin(fsPath); err != nil {
+		return nil, err
+	}
```

Comment:
Instead of adding another function `checkDiskTotalMin`, can't we reuse `checkDiskFree`?
like: checkDiskFree(fsPath, 1GB)